.bcr/core/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+  "integrity": "",
+  "strip_prefix": "{REPO}-{VERSION}/core",
+  "url": "https://github.com/{OWNER}/{REPO}/releases/download/{TAG}/${REPO}-${VERSION}.tar.gz"
```

Comment:
why not use `https://github.com/tweag/rules_nixpkgs/archive/refs/tags/v0.10.0.tar.gz` which is available by default?

Reply:
The short version is that these automatically generated artifacts are not guaranteed to be bit-wise reproducible. But, Bazel uses sha256 to ensure that the downloaded artifact was not tampered with or otherwise damaged. So, any bit-change in the artifact would change the sha256 and cause a failure. So, instead, the recommendation is that Bazel rules produce dedicated release artifacts, which are guaranteed to stay the same.
See also the discussion here: https://github.com/bazel-contrib/SIG-rules-authors/issues/11

Reply:
awesome, thank you!